### ISSUE TITLE
I encounter some bugs during building the program and integration in cadence and wonder if there lack some necessary file or definiton

### ISSUE THREAD

**Issue by zzhhjj-cpu** [patrickschulz/openPCells#11]
First , I have made the program and pointed my PATH to the directory, the result is as follows: please see Fig 1 &2 . When I type “opc --technology OPC_BIN --export svg --cell basic/transistor” in the command line, then there will be Fig 3，which gives “error while loading user config”.So I wonder if there lacks some definiton and maybe the program needs updates.

Second, after my friend and I discovered and supplemented the lacked definition above(see Fig 4 ), there still exist the following problem called "core dumped",please see Fig 5. We think the reason is that the corresponding file(home/.opcconfig.lua) is lacked 
,which needs to be supplemented with dictionary variables in the form of lua.(see Fig 6)
![image](https://user-images.githubusercontent.com/67616234/188256105-cde165c8-b980-4097-9df5-3381110a4bfd.png)
Fig 1
![image](https://user-images.githubusercontent.com/67616234/188256112-dca4ee99-a7df-49ac-a3ac-c0f4e6d7f402.png)
Fig 2
![image](https://user-images.githubusercontent.com/67616234/188256161-d90de3ea-dfdb-452f-b831-1e5fe05aaeb2.png)
Fig 3
![image](https://user-images.githubusercontent.com/67616234/188256556-e6be246b-d7bf-43a1-a61a-c419fe11afca.png)
Fig 4
![image](https://user-images.githubusercontent.com/67616234/188256370-6cc5c16f-7fa6-4853-87a0-18ef18e2cbf1.png)
Fig 5
![image](https://user-images.githubusercontent.com/67616234/188256579-57a4cc7f-155c-4f4c-ae06-54bbcf2fe547.png)
Fig 6.

Thanks for you opc!

**Comment by patrickschulz** [comment]
Hi and thank you for this report, this is indeed an error. A non-existing user config should not produce an error. I fixed this in commit https://github.com/patrickschulz/openPCells/commit/4fbf4a0bf5af49ad783d56163d44daee1d8fed48.
I did not exactly understand the second part of your issue. You created a config but still get a segmentation fault? I can't reproduce this by using `opc --cell basic/cmos --parameters`.
